shared/src/packages/orders/libs/constants/order-statuses.ts

```diff
@@ -0,0 +1,8 @@
+import { OrderStatus } from '../enums/order-status.enum.js';
+
+const ORDER_STATUSES = Object.values(OrderStatus as Record<number, string>) as [
+  string,
+  ...string[],
+];
+
```

Comment:
In my last comment I proposed like this, why don't you change?
```suggestion
import { OrderStatus } from '../enums/order-status.enum.js';

const ORDER_STATUSES = Object.values(OrderStatus);

```

Reply:
When I tried to do it in this way in line `const orderStatus = pgEnum('order_status', ORDER_STATUSES);` I get `Argument of type '("pending" | "confirmed" | "canceled" | "done")[]' is not assignable to parameter of type 'readonly [string, ...string[]] | [string, ...string[]]'.
  Type '("pending" | "confirmed" | "canceled" | "done")[]' is not assignable to type '[string, ...string[]]'.`